test/lib/BedrockTester.cpp

```diff
@@ -553,4 +553,38 @@ int BedrockTester::waitForPort(int port) {
     return 1;
 }
 
+// get the output of a "Status" command from the command port
```

Comment:
i'd put all of these function describing comments in the header file and remove them from the implementation file